src/overlays/actors/ovl_En_Md/z_en_md.h

```diff
@@ -29,18 +29,21 @@ typedef enum EnMdLimb {
 
 typedef void (*EnMdActionFunc)(struct EnMd*, PlayState*);
 
+#define ENMD_GET_PATH_INDEX(this) PARAMS_GET_S(this->actor.params, 8, 8)
+#define ENMD_PATH_IS_NONE(this) (PARAMS_GET_NOSHIFT(this->actor.params, 8, 8) == (NBITS_TO_MASK(8) << 8))
+
```

Comment:
For this, I meant to have `ENMD_GET_PATH_NOSHIFT` and `ENMD_PATH_NONE` as a separate define, like you had it before. 
In the scene file, we are still going to need to use `ENMD_PATH_NONE` as a standalone value when setting params. for example this one:
`    { ACTOR_EN_MD,  {      3,     48,    134 }, {      0, 0X8000,      0 }, 0xFF00 },`
will eventually be 
`    { ACTOR_EN_MD,  {      3,     48,    134 }, {      0, 0X8000,      0 }, ENMD_PARAMS(ENMD_PATH_NONE) },`

Reply:
Though this reminds me that the NONE define should probably just be 0xFF, not pre shifted. So not sure what to do about the NOSHIFT accesses. I guess just shift them in-place, idk

Reply:
alright, restored them as separate defines

Reply:
I think my above comment was missed, we're probably going to want to make PATH_NONE as just 0xFF. The value as is will get packed by the (eventual) ENMD_PARAMS macro.

But I am not sure how to handle it in conditions, other than doing 
```c
if (ENMD_GET_PATH_INDEX_NOSHIFT(this) == ENMD_PATH_NONE << 8) {
```

Reply:
aye, sorry, i have read it just a moment later. already updated it =)